### PR TITLE
🚑️ Update DMC to MC_X motors

### PR DESCRIPTION
**Pull request by FluffyFTW**
- Update the DMC motors to MC_X motors
- Changed i2c line to 1 for the pca9685
- Updated ESP to connect to AWS server

### CODE REVIEW

**Review by Coreyboy1820 — APPROVED** [review]

File: code/applications/control_loops/drive_serial.cpp [hunk 1]
```diff
@@ -27,9 +27,9 @@ hal::status application(sjsu::hardware_map& p_map)
 
   auto& terminal = *p_map.terminal;
   auto& clock = *p_map.steady_clock;
-  auto& magnet0 = *p_map.in_pin0;
-  auto& magnet1 = *p_map.in_pin1;
-  auto& magnet2 = *p_map.in_pin2;
+  // auto& magnet0 = *p_map.in_pin0;
+  // auto& magnet1 = *p_map.in_pin1;
+  // auto& magnet2 = *p_map.in_pin2;
```
Comment (Coreyboy1820) [inline code/applications/control_loops/drive_serial.cpp]: Uncomment this and all other motors.

**Review by Coreyboy1820 — CHANGES_REQUESTED** [review]
Just uncomment the items I told you to and merge it in.

**Review by Coreyboy1820 — APPROVED** [review]

**Review by Coreyboy1820 — CHANGES_REQUESTED** [review]
Change the arm code to have the correct esp connection loop.

**Comment by kammce** [comment]
Were you able to upgrade the MC_X firmware?

**Pull request closed without merging**